assets/src/components/v2/departures/header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from "react";
+import cx from "classnames";
+import { imagePath } from "Util/util";
+
+type CardinalDirection = "n" | "ne" | "e" | "se" | "s" | "sw" | "w" | "nw";
+
+interface Props {
+  title: string | null;
+  direction: CardinalDirection | null;
+}
+
+const DirectionArrow = ({ direction }: { direction: CardinalDirection }) => {
+  const src = useMemo(() => imagePath("arrow-45.svg"), []);
+  return (
+    <img
+      src={src}
+      data-direction={direction}
+      className={cx("direction-arrow", {
+        "rotate-45": direction === "n",
+        "rotate-90": direction === "ne",
+        "rotate-135": direction === "e",
+        "rotate-180": direction === "se",
+        "rotate-225": direction === "s",
+        "rotate-270": direction === "sw",
+        "rotate-315": direction === "w",
+      })}
+    />
+  );
+};
+
+const Header = ({ title, direction }: Props) => {
+  return (
+    <header className="departures-header">
+      {(title || direction) && <span>{title}</span>}
+      {direction && <DirectionArrow direction={direction} />}
+    </header>
+  );
+};
```

Comment:
**note:** I believe the intent is for a header with a `read_as` value but no `title` to be audio only. If that's not the case this will need to be updated to fallback to `read_as` if `title` is not set.

Reply:
Correct. In reality I don't think we have any such screens currently (if an audio version of the header is configured, it always has a visual title as well), so we're free to decide how we want this to work. And I can see audio-only headers being useful in scenarios where we don't need the wayfinding but do want an audio "separator" between sections to match the visual separator that is still present when there's no title.